Readability.js

```diff
@@ -680,6 +688,7 @@ Readability.prototype = {
         if (stripUnlikelyCandidates) {
           if (this.REGEXPS.unlikelyCandidates.test(matchString) &&
               !this.REGEXPS.okMaybeItsACandidate.test(matchString) &&
+              !this._isMaybeAncestorACandidate(node, 3) &&
```

Comment:
I use 3 here because we use the 3 number in a other place, `var ancestors = this._getNodeAncestors(elementToScore, 3);`[1]. Seems 3 level depth means something for ancestors, and I'm also wondering why 3 level depth for ancestors?

[1]: https://github.com/mozilla/readability/blob/master/Readability.js#L741